nuts/plugin.py

```diff
@@ -73,7 +73,7 @@ def pytest_collect_file(parent: Session, path: LocalPath) -> Optional[Collector]
     return None
 
 
-def pytest_addoption(parser):
+def pytest_addoption(parser: Any) -> None:
     """Add pytest command line options to configure nuts"""
```

Comment:
According to [pytest's documentation](https://docs.pytest.org/en/6.2.x/reference.html) this is of type `Parser`: `pytest_addoption(parser: Parser, pluginmanager: PytestPluginManager) `

Reply:
Done, thanks for the hint.